style.css

```diff
@@ -0,0 +1,17 @@
+#main {
+  background-color: black;
+  font-family: arial, helvetica, sans-serif;
+  color: white;
+  text-align: center;
```

Comment:
Please stick to the design as much as possible (e.g., font, colours, font-family and margins)

Reply:
Please always includes an explanation, a "why", for each requested change. For example:

This is a well-explained request: Please stick to the design as much as possible (e.g., font, colors, font family, and margins) as a mandate in the project requirement 👍 

This request lacks a "why": Please stick to the design as much as possible (e.g., font, colors, font-family, and margins)